project/dexter-core/src/java/com/samsung/sec/dexter/core/analyzer/AnalysisConfig.java

```diff
@@ -184,6 +193,15 @@ public void setHeaderBaseDirList(final List<String> baseHeaderFolderList) {
 		this.headerBaseDirList = baseHeaderFolderList;
 	}
 
+	
+	public final List<String> getFunctionList(){
```

Comment:
If your intention is the immutable object of functionList member, this does not work.
But if you want to prevent overriding, this is fine.